core/tests/tests/Validation.mint

```diff
@@ -11,11 +11,33 @@ suite "Validation.isNotBlank" {
 suite "Validation.isNumber" {
   test "it returns nothing if the string can be converted to number" {
     Validation.isNumber("0", {"key", "ERROR"}) == Maybe::Nothing
+    Validation.isNumber("0.1", {"key", "ERROR"}) == Maybe::Nothing
```

Comment:
Line 13 and 14 will not count in the assertion. They need to be chained with `&&` or put into separate test cases.

Reply:
Thank you, I was not aware of that.